### ISSUE TITLE
ogr2ogr metrics inception

### ISSUE THREAD

**Issue by santisaez** [CartoDB/cartodb#3015]
`loader/ogr2ogr` metric is usually creating children metrics with the same name, example:

```bash
/var/lib/graphite/whisper/stats/timers/importer/$hostname/$cloud/cartodb/net/run/resource/import/loader/ogr2ogr/loader/ogr2ogr/loader/ogr2ogr/loader/ogr2ogr/loader/ogr2ogr/loader/ogr2ogr/loader/ogr2ogr/loader/ogr2ogr/loader/ogr2ogr/loader/ogr2ogr/loader/ogr2ogr/loader/ogr2ogr/loader/ogr2ogr/loader/ogr2ogr/loader/ogr2ogr/loader/ogr2ogr/loader/ogr2ogr/loader/ogr2ogr/loader/ogr2ogr/loader/ogr2ogr/loader/ogr2ogr/loader/ogr2ogr/loader/ogr2ogr/loader/ogr2ogr/loader/ogr2ogr/loader/ogr2ogr/loader/ogr2ogr/loader/ogr2ogr/loader/ogr2ogr/loader/ogr2ogr/loader/ogr2ogr/loader/ogr2ogr/loader/ogr2ogr/loader/ogr2ogr/loader/ogr2ogr/loader/ogr2ogr/loader/ogr2ogr/loader/ogr2ogr/loader/ogr2ogr/loader/ogr2ogr/loader/ogr2ogr/loader/ogr2ogr/loader/ogr2ogr/loader/ogr2ogr/loader/ogr2ogr/loader/ogr2ogr/loader/ogr2ogr/loader/ogr2ogr/loader/ogr2ogr/loader/ogr2ogr/resource/file_size_limit_check

(..)

# du -sh /var/lib/graphite/whisper/stats/timers/importer/$hostname/$cloud/cartodb/net/run/resource/import/loader/ogr2ogr
928M	/var/lib/graphite/whisper/stats/timers/importer/$hostname/$cloud/cartodb/net/run/resource/import/loader/ogr2ogr
```

Related: https://github.com/CartoDB/cartodb-platform/issues/369 and https://github.com/CartoDB/Windshaft-cartodb/issues/277

**Comment by juanignaciosl** [comment]
Duplicated: #2562 . I already tried a fix (a999916349ddbf64a4e2da8c53c0ca2307196661) but it seems to be insufficient.

@santisaez are there more examples? I'd like to know whether it's always related to `file_size_limit_check` or not.

**Comment by rafatower** [comment]
this shall be fixed elsewhere, otherwise feel free to reopen
